docs/basics/101-126-filesystem.rst

```diff
@@ -0,0 +1,422 @@
+   $ datalad status
+
+That's expected. The copy shows up as a new, untracked
+file. Let's save it:
+
+.. runrecord:: _examples/DL-101-114-131
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ datalad save -m "add copy of TLCL.pdf"
+
+.. runrecord:: _examples/DL-101-114-132
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ git log -1 -p
+
+That's it.
+
+.. container:: toggle
+
+   .. container:: header
+
+      **Addition: Symlinks!**
+
+   If you have read the additional content in the section
+   :ref:`symlinks`, you know that the same file content
+   is only stored once, and copies of the same file point to
+   the same location in the object tree.
+
+   Let's check that out:
+
+   .. runrecord:: _examples/DL-101-114-133
+      :language: console
+      :workdir: dl-101/DataLad-101
+
+      $ ls -l copyofTLCL.pdf
+      $ ls -l books/TLCL.pdf
+
+   Indeed! Apart from their relative location (``.git`` versus
+   ``../.git``) their symlink is identical. Thus, even though two
+   copies of the book exist in your dataset, your disk needs to
+   store only one.
+
+Finally, let's clean up:
+
+.. runrecord:: _examples/DL-101-114-134
+   :language: console
+   :workdir: dl-101/DataLad-101
+
+   $ git reset --hard HEAD~1
+
+What happens if I rename a directory or subdataset?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+What happens if I move directories, or subdatasets?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+TODO: directory (all files in a directory will be renamed)
+TODO: subdataset (tricky: change in .gitmodules. Also, I failed trying to
+revert a ``git mv`` with ``git reset --hard master``. It did not move
+subdataset back into original place, the subds became an untracked directory.
+
+What if I move or rename a superdataset?
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Once created, a DataLad superdataset may not be in an optimal
+place on your file system, or have the best name.
+
+After a while, you might think that the dataset would fit much
+better into ``/home/user/research_projects/`` than in
+``/home/user/Documentents/MyFiles/tmp/datalad-test/``. Or maybe at
+some point, a long name such as ``My-very-first-DataLad-project-wohoo-I-am-so-excited``
+does not look pretty in your terminal prompt anymore, and going for
+``finance-2019`` seems more professional.
+
+These will be situations in which you want to rename or move
+a superdataset. Will that break anything?
+
+In all standard situations, no, it will be completely fine.
+You can use standard Unix commands such as ``mv`` to do it,
+and also which ever graphical user interface or explorer you may
+use.
+
+Beware of one thing though: If your dataset either is a sibling
+or has a sibling with the source being a path, moving or renaming
+the dataset will break the linkage between the datasets. This can
+be fixed easily though. Let's try it.
+
+As section (todo: link section on config files) explains, each
```

Comment:
Given our conversation last week, I'm not sure whether this is a best practice.
TL;DR: Lets say I rename/move a dataset that is another datasets sibling -- is it okay to adjust the remote url in the .git/config file? It works, but is there a preferred way?